packages/scripts/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- Upgrade `puppeteer-core` (`^9.0.0`) to version `^10.1.0`. This version drops support for Node v10.
```

Comment:
https://github.com/WordPress/gutenberg/blob/d0a25e84180757a34d1ef1c989e91f8efc6dc368/packages/scripts/package.json#L21-L24

`@wordpress/scripts` doesn't support Node v10 for quite some time already.